storage/buckets/add_bucket_label.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package buckets
+
+// [START storage_add_bucket_label]
```

Comment:
I noticed you have add/remove from https://cloud.google.com/storage/docs/using-bucket-labels on your list but not storage_get_bucket_labels, which is also missing a Go sample. Is that an oversight on our part @frankyn @JesseLovelace ?

Reply:
Adding an excerpt from internal requirements document.
```
storage_get_bucket_metadata
- Should contain metadata for the following samples and don't use the following samples:
-- storage_get_bucket_labels
-- storage_view_lifecycle_management_configuration
-- storage_bucket_get_default_kms_key
-- storage_view_versioning_status
-- storage_print_bucket_website_configuration
-- storage_get_requester_pays_status
-- storage_get_cors_configuration
```